src/System.Net.Primitives/src/System/Net/CookieException.cs

```diff
@@ -10,13 +10,22 @@
 // System.Private.Networking dll. So, we need to move the classes to a different namespace. Those classes are never
 // exposed up to user code so there isn't a problem.  In the future, we might expose some of the internal methods
 // as new public APIs and then we can remove this duplicate source code inclusion in the binaries.
+using System;
+
+#if !netcore50
```

Comment:
I head something like that from @ericeil , but i dont know the latest.I will ask around to see what should be done